flixel/addons/editors/tiled/TiledObjectLayer.hx

```diff
@@ -0,0 +1,32 @@
+package flixel.addons.editors.tiled;
+
+import haxe.xml.Fast;
+import flixel.util.FlxColor;
+import flixel.addons.editors.tiled.TiledLayer.TiledLayerType;
+
+/**
+ * Copyright (c) 2013 by Samuel Batista
+ * (original by Matt Tuttle based on Thomas Jahn's. Haxe port by Adrien Fischer)
+ * This content is released under the MIT License.
+ */
+class TiledObjectLayer extends TiledLayer
+{
+	public var objects:Array<TiledObject>;
+	public var color:FlxColor;
+	
+	public function new(source:Fast, parent:TiledMap)
+	{
+		super(source, parent);
+		type = TiledLayerType.OBJECT;
+		objects = new Array<TiledObject>();
+		color = source.has.color ? FlxColor.fromString(source.att.color) : FlxColor.TRANSPARENT;
+		loadObjects(source);
+	}
+	
+	private function loadObjects(source:Fast):Void {
```

Comment:
`{` should be in newline here.

Man, there really needs to be an auto-formatter for Haxe. :/